code/globalincs/systemvars.cpp

```diff
@@ -245,57 +245,89 @@ detail_levels Detail_defaults[NUM_DEFAULT_DETAIL_LEVELS] = {
 // Global used to access detail levels in game and libs
 detail_levels Detail = Detail_defaults[NUM_DEFAULT_DETAIL_LEVELS - 1];
 
-const SCP_vector<std::pair<int, SCP_string>> DetailLevelValues = {{ 0, "Minimum" },
-                                                                  { 1, "Low" },
-                                                                  { 2, "Medium" },
-                                                                  { 3, "High" },
-                                                                  { 4, "Ultra" }, };
-
-const auto ModelDetailOption __UNUSED =
-	options::OptionBuilder<int>("Graphics.Detail", "Model Detail", "Detail level of models").importance(8).category(
-		"Graphics").values(DetailLevelValues).default_val(MAX_DETAIL_LEVEL).change_listener([](int val, bool) {
-		Detail.detail_distance = val;
-		return true;
-	}).finish();
+const SCP_vector<std::pair<int, std::pair<const char*, int>>> DetailLevelValues = {{ 0, {"Minimum", 1780}},
```

Comment:
This is probably a late observation, but usage of an int-string pair is an odd choice to associate an int value with a string when the position of the strings in its containing vector can suffice.  If this is a limitation of the ::values member and mechanisms using it, then please disregard.

Reply:
That is waaaay outside the scope of this PR and something part of the initial design of the OptionsBuilder Values that m!m wrote back in 2019. I'm not going to touch that here.